Explorer/Assets/DCL/AvatarRendering/AvatarShape/Systems/AvatarInstantiatorSystem.cs

```diff
@@ -113,7 +113,7 @@ private void InstantiateMainPlayerAvatar(in Entity entity, ref AvatarShapeCompon
             InstantiateNewAvatar(entity, ref avatarShapeComponent, ref transformComponent);
 
             if (World.TryGet(entity, out AvatarBase avatarBase))
-                mainPlayerAvatarBase.SetAvatarBase(avatarBase);
```

Comment:
Should it release the reference in InternalDestroyAvatar method as well?

Reply:
Added a check and release in the same "proxy" refactor at [this commit](https://github.com/decentraland/unity-explorer/pull/340/commits/e58d6c8346f2450f1f8bfa137c2e610704d62991)